### ISSUE TITLE
Code in examples/build-package is failing after #362 was merged

### ISSUE THREAD

**Issue by antonbabenko** [terraform-aws-modules/terraform-aws-lambda#363]
## Description

As described in [this comment](https://github.com/terraform-aws-modules/terraform-aws-lambda/pull/311#issuecomment-1287864173), examples are failing.

## Reproduction Steps

```
git checkout master
cd examples/build-package
terraform init
terraform apply
```

Error is like this:
```
│     raise RuntimeError(
│ RuntimeError: Nodejs interpreter version equal to defined lambda runtime (nodejs14.x) should be available in system PATH
```

## Expected behavior

Examples should work as long as npm, pip, docker is installed.

/cc @lorengordon Could you take a look at this issue, please?

It is currently blocking #311 from testing and merging.

**Comment by lorengordon** [comment]
Ahh, I don't use nodejs, so I misinterpreted the apply-time code. The error message there _says_ the "runtime" must be available in the PATH, but really it is `npm` that must be in the PATH...

https://github.com/terraform-aws-modules/terraform-aws-lambda/blob/fa92a3992b9c218d0770d822131dcc1a0689a34a/package.py#L1036

I can't get to it immediately, and tomorrow is unlikely, but maybe sometime Monday.

**Comment by antonbabenko** [comment]
@lorengordon Monday is fine, too. Thank you very much! Have a great weekend!

**Comment by vemulkri** [comment]
seeing similar error, just posting for visibility. Thanks

`RuntimeError: Python interpreter version equal to defined lambda runtime
(python3.8) should be available in system PATH`

**Comment by lorengordon** [comment]
@vemulkri Do you have `python3.8` in your PATH? The problem this issue is about, is that it is looking for the command `nodejs14.x` but the command is `npm`. However, for python, it should be the actual runtime, so `python3.8` is the correct value.

**Comment by lorengordon** [comment]
@antonbabenko Please see #364

**Comment by vemulkri** [comment]
> @vemulkri Do you have `python3.8` in your PATH? The problem this issue is about, is that it is looking for the command `nodejs14.x` but the command is `npm`. However, for python, the command should be the actual runtime, so `python3.8` is the correct value.

@lorengordon Apologies, added specific runtime to path of my github actions and working as expected now. Thanks.

**Comment by antonbabenko** [comment]
This issue has been resolved in version 4.2.2 :tada:
